mirdata/datasets/idmt_smt_audio_effects.py

```diff
@@ -0,0 +1,287 @@
+    plugins.
+
+    To organize the database, lists in XML format are used, which record all relevant information and are provided with
+    the database as well as a summary of the used effect plugins and parameter settings.
+
+    In addition, most of this information is also encoded in the first part of the file name of the audio files using 
+    a simple alpha-numeric encoding scheme. The second part of the file name contains unique identification numbers. 
+    This provides an option for fast and flexible structuring of the data for various purposes.
+
+    DOI
+    10.5281/zenodo.7544032
+"""
+import os
+import librosa
+import numpy as np
+import xml.etree.ElementTree as ET
+
+from deprecated.sphinx import deprecated
+from typing import BinaryIO, Tuple, Optional
+from mirdata import download_utils, jams_utils, core, io
+from smart_open import open
+
+BIBTEX = """
+@dataset{stein_michael_2023_7544032,
+  author       = {Stein, Michael},
+  title        = {IDMT-SMT-Audio-Effects Dataset},
+  month        = jan,
+  year         = 2023,
+  publisher    = {Zenodo},
+  version      = {1.0.0},
+  doi          = {10.5281/zenodo.7544032},
+  url          = {https://doi.org/10.5281/zenodo.7544032}
+}
+"""
+
+INDEXES = {
+    "default": "1.0",
+    "test": "1.0",
+    "1.0": core.Index(filename="idmt_smt_audio_effects_index.json"),
+}
+
+REMOTES = {
+    "full_dataset": download_utils.RemoteFileMetadata(
+        filename="IDMT-SMT-AUDIO-EFFECTS.zip",
+        url="https://zenodo.org/record/7544032/files/IDMT-SMT-AUDIO-EFFECTS.zip?download=1",
+        checksum="91e845a1b347352993ebd5ba948d5a7c",  # the md5 checksum
+        destination_dir=".",  # relative path for where to unzip the data, or None
+        unpack_directories=[""],
+    ),
+}
+
+DOWNLOAD_INFO = """
+        This loader will create the following folders in the dataset data_home path:
+            > idmt_smt_audio_effects/
+                > Bass monophon/
+                > Bass monophon2/
+                > Gitarre monophon/
+                > Gitarre monophon2/
+                > Gitarre polyphon/
+                > Gitarre polyphon2/
+"""
+
+LICENSE_INFO = """
+Creative Commons BY-NC-ND 4.0.
+https://creativecommons.org/licenses/by-nc-nd/4.0/
+"""
+
+
+class Track(core.Track):
+    """IDMT-SMT-Audio-Effects track class.
+
+    Args:
+        track_id (str): track id of the track.
+        data_home (str): Local path where the dataset is stored.
+        dataset_name (str): Name of the dataset.
+        index (Dict): Index dictionary.
+        metadata (Dict): Metadata dictionary.
+
+    Attributes:
+        audio_path (str): path to audio file.
+
+    Cached Properties:
+        instrument (str): instrument used to record the track.
+        midi_nr (int): midi number of the note.
+        fx_group (int): effect group number.
+        fx_type (int): effect type number.
+        fx_setting (int): effect setting number.
```

Comment:
These are  not `Cached Properties`, right...? I think they should be documented as `Attributes` here.